app/src/main/java/com/example/lhi06/residue/Album.java

```diff
@@ -4,9 +4,9 @@
 
 public class Album {
 
-    private final String collectionName;
-    private final String artistName;
-    private final String artworkUrl100;
+    private String collectionName;
+    private String artistName;
+    private String artworkUrl100;
```

Comment:
@david-williams-sky do you think it's worth introducing [SerialisedName](https://google.github.io/gson/apidocs/com/google/gson/annotations/SerializedName.html)?